packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -210,7 +224,13 @@ export const connectEpic: Epic<AnyAction, AnyAction, any, any> = (action$, state
       )
 
       const unresponsiveOnTimeout$ = messageReceived$.pipe(
-        mergeMap(({ payload: { input, subscriptionKey } }) => {
+        withLatestFrom(state$),
+        map(([action, state]) => {
+          const input = state.ws.subscriptions[action.payload.subscriptionKey]?.input || {}
+          if (!input) logger.warn(`WS: Could not find subscription from incoming message`)
+          return { ...action, input }
+        }),
```

Comment:
Same as above, just fetch the input cleanly in the `mergeMap` below, instead of passing it with an action.